ocaml/xapi/xapi_cluster.ml

```diff
@@ -151,7 +153,7 @@ let pool_destroy_common ~__context ~self ~force =
         List.filter (( <> ) master_ch) all_hosts
   in
   foreach_cluster_host ~__context ~self ~log:force
-    ~fn:Client.Client.Cluster_host.destroy slave_cluster_hosts
```

Comment:
There is another instance of `slave` here

Reply:
Yes, there are a lot of mentions to slaves around the whole codebase. I concentrated the efforts of this PR around xapi_pool_helpers and its users. I can extend it a bit more if we find the terminology good enough to use.